tests/homework18/homework18.spec.js

```diff
@@ -0,0 +1,207 @@
+    const passwordError = page.locator('div.invalid-feedback p');
+    await passwordInput.fill('test.email',{ timeout: 2000 });
+    await passwordInput.focus();
+    await passwordInput.blur();
+    await expect(passwordError).toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter');  });
+
+  test('Invalid Password data - Too short', async ({ page }) => {
+    const passwordInput = page.locator('input[name="password"]');
+    const passwordError = page.locator('div.invalid-feedback p');
+    await passwordInput.fill('Aa', {timeout: 2000});
+    await passwordInput.focus();
+    await passwordInput.blur();
+    await expect(passwordError).toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter');  });
+
+test('Invalid Password data - Too long', async ({ page }) => {
+  const passwordInput = page.locator('input[name="password"]');
+  const passwordError = page.locator('div.invalid-feedback p');
+  await passwordInput.fill('A'.repeat(20));
+  await passwordInput.focus();
+  await passwordInput.blur();
+  await expect(passwordError).toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter');
+});
+
+test('Invalid Password data - No capital letter', async ({ page }) => {
+  const passwordInput = page.locator('input[name="password"]');
+  const passwordError = page.locator('div.invalid-feedback p');
+  await passwordInput.fill('a'.repeat(6));
+  await passwordInput.focus();
+  await passwordInput.blur();
+  await expect(passwordError).toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter');
+});
+
+test('Invalid Password data - No small letter', async ({ page }) => {
+  const passwordInput = page.locator('input[name="password"]');
+  const passwordError = page.locator('div.invalid-feedback p');
+  await passwordInput.fill('A1'.repeat(6));
+  await passwordInput.focus();
+  await passwordInput.blur();
+  await expect(passwordError).toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter');
+});
+
+test('Empty Re-enter Password field validation', async ({ page }) => {
+  const repeatPasswordInput = page.locator('input[name="repeatPassword"]');
+  const secondPasswordError = page.locator('div.invalid-feedback p');
+  await repeatPasswordInput.fill('');
+  await repeatPasswordInput.focus();
+  await repeatPasswordInput.blur();
+  await expect(secondPasswordError).toHaveText('Re-enter password required');
+});
+
+test('Invalid Re-enter Password data - Passwords do not match', async ({ page }) => {
+  const repeatPasswordInput = page.locator('input[name="repeatPassword"]');
+  const secondPasswordError = page.locator('div.invalid-feedback p');
+  await repeatPasswordInput.fill('passWord1');
```

Comment:
do not hardcode values

Reply:
const newPassword = 'passWord1';
  await repeatPasswordInput.fill(newPassword);